templates/base.html

```diff
@@ -35,6 +35,15 @@
 			</div>
 			{% block search %}{% endblock %}
 			<div id="links">
+				{% if prefs.theme == "" || prefs.theme == "light" %} 
```

Comment:
If theme is unset, or is light

This could use a more involved check about if the theme is unset but the user has dark mode enabled (and as such, should be shown the light button)